uobtheatre/productions/models.py

```diff
@@ -681,6 +705,16 @@ class Meta:
         ordering = ["id"]
 
 
+def delete_user_drafts(user, performance_id, booking_id=None):
+    """Remove's the users exisiting draft booking for the given performance"""
```

Comment:
Should include proper docs for this function (i.e. what do `performance_id` and `booking_id` represent)